remote-config.jsonc

```
@@ -198,14 +201,54 @@
           "message": "Platform.sh is the lead sponsor of DDEV, paying Randy's salary! Stop by and thank them at https://platform.sh."
         },
         {
-          "message": "A key DDEV goal this year is to train and fund a third paid maintainer."
+          "message": "A key DDEV goal for 2024 is to train and fund a third paid maintainer."
         },
         {
           "message": "Take a moment to read the DDEV 2023 review: https://ddev.com/blog/2023-review"
         },
         {
           "message": "It's the DDEV community that makes DDEV what it is. THANK YOU!"
         },
+        {
+          "message": "The DDEV Foundation is now an official US 501(c)(3) organization, meaning it doesn't have to pay taxes and contributions to it are tax-deductible. https://ddev.com/blog/501c3/"
+        },
+        {
+          "message": "Have you ever wondered how 'ddev.site' domain names work? Read all about it at https://ddev.com/blog/ddev-name-resolution-wildcards"
+        },
+        {
+          "message": "Use 'ddev_version_constraint' in your .ddev/config.yaml to make sure all your team members are on a minimum DDEV version, for example `ddev_version_constraint: '>=v1.23.0'`"
+        },
+        {
+          "message": "If you use the new 'drupal' or 'cakephp' project types for your project, use `ddev_version_constraint: '>=v1.23.0'` to make sure your team members have a version that supports them."
+        },
+        {
+          "message": "DDEV v1.23.0 ddev-webserver is based on Debian 12 Bookworm. If you use custom webimage_extra_packages or .ddev/web-build/Dockerfile you may have to adjust them. For example, the old 'python' package is gone (but python3 is already there for you)."
+        },
+        {
+          "message": "DDEV is happy to help you with settings management, for example by creating a settings.ddev.php, but it's just there to make it easier for you. Turn it off with `ddev config --disable-settings-management', more in https://ddev.readthedocs.io/en/stable/users/usage/cms-settings/"
+        },
+        {
+          "message": "Are you using DDEV in CI or other automated testing? If so, please turn off instrumentation, `ddev config global --instrumentation-opt-in=false` or `export DDEV_NO_INSTRUMENTATION=true`"
```

```suggestion
          "message": "Are you using DDEV in Continuous Integration (CI) or any kind of automated testing? If so, please turn off instrumentation, `ddev config global --instrumentation-opt-in=false` or `export DDEV_NO_INSTRUMENTATION=true`"
```
i just try to avoid abbreviations (in particular for stand alone tipps it makes sense otherwise it might require the person reading to look it up) - we had an extensive discussion in the aus-nz channel in the drupal slack about that topic)